sdk/storage/storage-file-share/README.md

```diff
@@ -102,6 +100,15 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
+## Key concepts
+
+The following components and their corresponding client libraries make up the Azure Storage File Share service:
+
+- The storage account itself, represented by a `ShareServiceClient`
```

Comment:
In other places, the _resources name_ is in italic.  Better to be consistent here too.